nemo/collections/llm/peft/lora.py

```diff
@@ -258,9 +259,6 @@ class LoRA(PEFT):
     )
     """
 
-    target_modules: List[str] = field(
-        default_factory=lambda: ['linear_qkv', 'linear_proj', 'linear_fc1', 'linear_fc2']
-    )
```

Comment:
i think we can still keep this, so that user knows what's set as default when they just visit this class